Automattic/jetpack
# Sync: Update how we sync User data

If applied, this diff will:
- change the order of sync actions for user-related events by delaying certain syncs to happen after wp_insert_user
- create a new sync event for when a user's role is changed.

Still in progress, Unit tests need updating.

To test:
With this diff applied try
- adding a new user
- editing a user
- adding / editing via API
- editing your profile
- add existing user to multi-site
- invite user -> user accepts invite
- user registers via registration form
- delete user

Ensure stuff is syncing properly upstream.
